src/Illuminate/Routing/Router.php

```diff
@@ -236,13 +236,27 @@ public function fallback($action)
      * @param  int  $status
      * @return \Illuminate\Routing\Route
      */
-    public function redirect($uri, $destination, $status = 301)
+    public function redirect($uri, $destination, $status = 302)
     {
         return $this->any($uri, '\Illuminate\Routing\RedirectController')
                 ->defaults('destination', $destination)
                 ->defaults('status', $status);
     }
 
+    /**
+     * Create a permanent redirect from one URI to another.
+     *
+     * @param  string  $uri
+     * @param  string  $destination
+     * @return \Illuminate\Routing\Route
+     */
+    public function permanentRedirect($uri, $destination)
+    {
+        return $this->any($uri, '\Illuminate\Routing\RedirectController')
```

Comment:
Just `return $this->redirect($uri, $destination, 301);`?

Reply:
You are right. Fixed ;)